examples/6.educational_pluto_notebook/simulation-example.jl

```diff
@@ -245,70 +243,54 @@ The spin echo experiment has the advantage that the echo signal amplitud it is m
- - (4.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in $$\mathrm{TE}$$
- - (4.5) Create concatenating these blocks into a sequence called `seq_se`
- - (4.6) Plot `seq_se` and its k-space. Is the k-space the same as `seq_gre` in (2.3)?
+- (4.1) Create a sequence called `seq_se` concatenating the following blocks:
+  * (4.1.1) A 90deg hard RF pulse
+  * (4.1.2) A `Delay` of $$\mathrm{TE}/2$$ with a positive gradient (area `Ax`)
+  * (4.1.3) A 180deg hard RF pulse
+  * (4.1.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in $$\mathrm{TE}$$
+ 
+ - (4.2) Plot `seq_se` and its k-space. Is the k-space the same as `seq_gre` in (2.3)?
 """
 
 # ╔═╡ c8a37593-3028-4e50-ad07-dc81edba45c8
-# (4.1) A 90deg hard RF pulse
-# ...
-
-# (4.2) A `Delay` of TE/2 with a positive gradient (area `Ax`)
+# (4.1) Create a Spin Echo sequence `seq_se`
+# (4.1.1) A 90deg hard RF pulse
 # ...
-
-# (4.3) A 180deg hard RF pulse
+# (4.1.2) A `Delay` of TE/2 with a positive gradient (area `Ax`)
 # ...
-
-# ╔═╡ 75ee6dbe-598a-47be-9655-3de3bc015281
-# (4.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in TE
+# (4.1.3) A 180deg hard RF pulse
 # ...
-
-# ╔═╡ 68d88987-3de7-42ae-9380-91ffae3ca40b
-# (4.5) Create concatenating these blocks into a sequence called `seq_se`
+# (4.1.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in TE
 # ...
 
 # ╔═╡ f1f3b700-5916-496f-b938-46f7f08b4eb6
-# (4.6) Plot seq_se and its k-space. Is the k-space the same as seq_gre in (2.3)?
+# (4.2) Plot seq_se and its k-space. Is the k-space the same as seq_gre in (2.3)?
 # ...
 
 # ╔═╡ 45952512-aaf1-43d8-a95e-c32bb2633f42
 md"""
- - (4.7) Simulate using `seq_se` and `obj_t2star`
- - (4.8) Compare the signal obtained in (4.6) with the one at (3.5)
- - (4.9) Reconstruct the 1D image
+ - (4.3) Simulate using `seq_se` and `obj_t2star`
+ - (4.4) Compare the signal obtained in (4.6) with the one at (3.5)
+ - (4.5) Reconstruct the 1D image
 """
 
 # ╔═╡ 97479437-9ce3-4b33-9134-0f2af89bccb5
-# (4.7) Simulate using seq_se and obj_t2star
+# (4.3) Simulate using seq_se and obj_t2star
 # ...
 
 # ╔═╡ 1c79b37e-d4e0-490f-9466-20ce28f017ae
-# (4.8) Compare the signal obtained in (4.6) with the one at (3.5)
+# (4.4) Compare the signal obtained in (4.6) with the one at (3.5)
 # ...
 
 # ╔═╡ 2e65ae31-f50a-462b-9744-80bf6cdb388e
-# (4.9) Reconstruct the 1D image
+# (4.5) Reconstruct the 1D image
 # ...
 
 # ╔═╡ fe8bbcd2-e8f5-4225-80c3-47e73176fb3d
 md"""
 Congratulations! you finished the simulation hands-on session 🥳!
 """
 
-# ╔═╡ 3b6b91cf-f3ad-40bc-9b3b-8bb5f395537f
-# Run this cell to celebrate!
-html"""
-<script>
-const {default: confetti} = await import("https://cdn.skypack.dev/canvas-confetti@1")
-confetti()
-</script>
-"""
-
```

Comment:
Why did you remove the confetti!? maybe add `activity_finished = false` and `if activity_finished html"""...""" end`.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/253/commits/3fde402ca7a7613045f15f2c77564a4f5491fcfb